.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
open to updating to a more strict config, doesn't have to be immediate. i did include @kelvin-lu sort-imports plugin.